core/arch/arch.h

```diff
@@ -1377,7 +1377,12 @@ typedef struct _callee_info_t {
     app_pc bwd_tgt;    /* earliest backward branch target */
     app_pc fwd_tgt;    /* last forward branch target */
     int num_simd_used; /* number of SIMD registers (xmms) used by callee */
-    bool simd_used[MCXT_NUM_SIMD_SLOTS]; /* SIMD (xmm/ymm) registers usage */
+
+    bool simd_used[MCXT_NUM_SIMD_SLOTS]; /* SIMD (xmm/ymm) registers usage. Part of
+                                          * the array might left uninitialized if
```

Comment:
grammar: might be

suggest: Move the comment to whole-line before the field, since it's large and doesn't really fit on the right side.

Reply:
Both changed.